templates/entity/view/entity.jade

```diff
@@ -13,6 +13,16 @@ mixin begin_end
 		dd=(entity.endDate ? entity.endDate : '?')
 
 block body
+	if alert
+		.row
+			.col-md-12
+				.alert.alert-info
+					div
+						strong Achievement unlocked:
+						each achievement in alert
+							p #{achievement.name}
+					div
+						strong Go to your profile to display
```

Comment:
I think this guideline is a bit confusing - if I were a new user, I'd think "display what?". Maybe showcase would be a better word? I don't know, try to think of a different way of saying it! :)